app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -117,85 +119,87 @@ enum class KeyboardLayout(val index: Int, val keyboardDefinition: KeyboardDefini
-    TRThumbKey(35, KB_TR_THUMBKEY),
-    ENTypeSplit(36, KB_EN_TYPESPLIT),
-    ESTypeSplit(37, KB_ES_TYPESPLIT),
-    DETypeSplit(38, KB_DE_TYPESPLIT),
-    FRTypeSplit(39, KB_FR_TYPESPLIT),
-    ITTypeSplit(40, KB_IT_TYPESPLIT),
-    PTTypeSplit(41, KB_PT_TYPESPLIT),
-    PLTypeSplit(42, KB_PL_TYPESPLIT),
-    ENTwoHands(43, KB_EN_TWO_HANDS),
-    ENThumbKeyProgrammerWide(44, KB_EN_THUMBKEY_PROGRAMMER_WIDE),
-    HUThumbKey(45, KB_HU_THUMBKEY),
-    ESEOThumbKey(46, KB_ES_EO_THUMBKEY),
-    ITMessageEase(47, KB_IT_MESSAGEEASE),
-    ENSKThumbKey(48, KB_EN_SK_THUMBKEY),
-    HEThumbKey(49, KB_HE_THUMBKEY),
-    EOENDEThumbKey(50, KB_EOENDE_THUMBKEY),
-    GRThumbKey(51, KB_GR_THUMBKEY),
-    ENCZThumbKey(52, KB_EN_CZ_THUMBKEY),
-    ESMessageEase(53, KB_ES_MESSAGEEASE),
-    RUMessageEase(54, KB_RU_MESSAGEEASE),
-    BGThumbKeySymbols(55, KB_BG_THUMBKEY_SYMBOLS),
-    HRTwoHands(56, KB_HR_TWO_HANDS),
-    HRThumbKey(57, KB_HR_THUMBKEY),
-    HRThumbKeySymbols(58, KB_HR_THUMBKEY_SYMBOLS),
-    FITypeSplit(59, KB_FI_TYPESPLIT),
-    LVLTGThumbKey(60, KB_LV_LTG_THUMBKEY),
-    LTThumbKey(61, KB_LT_THUMBKEY),
-    IDThumbKeySymbolsV2(62, KB_ID_THUMBKEY_SYMBOLS_V2),
-    IDThumbKeySymbolsNumeric(63, KB_ID_THUMBKEY_SYMBOLS_NUMERIC_V1),
-    ESCAThumbKey(64, KB_ES_CA_THUMBKEY),
-    ENITThumbKey(65, KB_EN_IT_THUMBKEY),
-    ENEOMessageEaseSymbols(66, KB_ENEO_MESSAGEEASE_SYMBOLS),
-    UKRUMessageEaseSymbols(67, KB_UK_RU_MESSAGEEASE_SYMBOLS),
-    DEMessageEaseSymbols(68, KB_DE_MESSAGEEASE_SYMBOLS),
-    CAThumbKey(69, KB_CA_THUMBKEY),
-    MATHThumbKey(70, KB_MATH_THUMBKEY),
-    ENEEThumbKey(71, KB_EN_EE_THUMBKEY),
-    SKThumbKey(72, KB_SK_THUMBKEY),
-    ENNOTypeSplit(73, KB_EN_NO_TYPESPLIT),
-    ENThumbKeyCompose(74, KB_EN_THUMBKEY_COMPOSE),
-    ENNOThumbKey(75, KB_EN_NO_THUMBKEY),
-    ENTwoHandsSymbols(76, KB_EN_TWO_HANDS_SYMBOLS),
-    ENTwoHandsSymbolsNumbers(77, KB_EN_TWO_HANDS_SYMBOLS_NUMBERS),
-    JATypeSplitHiragana(78, KB_JA_TYPESPLIT_HIRAGANA),
-    JATypeSplitKatakana(79, KB_JA_TYPESPLIT_KATAKANA),
-    EUESThumbKey(80, KB_EU_ES_THUMBKEY),
-    ENQwertyFour(81, KB_EN_QWERTYFOUR),
-    KZThumbKey(82, KB_KZ_THUMBKEY),
-    ENMIThumbKey(83, KB_EN_MI_THUMBKEY),
-    ENMIThumbKeySymbols(84, KB_EN_MI_THUMBKEY_SYMBOLS),
-    ESCAMessageEase(85, KB_ES_CA_MESSAGEEASE),
-    SVMessageEase(86, KB_SV_MESSAGEEASE),
-    ENDoubleSymbols(87, KB_EN_DOUBLE_SYMBOLS),
-    FIMessageEase(88, KB_FI_MESSAGEEASE),
-    ENDAThumbKeySymbols(89, KB_EN_DA_THUMBKEY_SYMBOLS),
-    FIEEMessageEase(90, KB_FI_EE_MESSAGEEASE_SYMBOLS),
-    ENPLThumbkey(91, KB_EN_PL_THUMBKEY),
-    BGPhoneticMessagEaseSymbols(92, KB_BG_PHONETIC_MESSAGEASE_SYMBOLS),
-    UKMessageEaseSymbols(93, KB_UK_MESSAGEEASE_SYMBOLS),
-    ENFRMessagEaseSymbols(94, KB_EN_FR_MESSAGEASE_SYMBOLS),
-    HEMessageEaseSymbols(95, KB_HE_MESSAGEEASE_SYMBOLS),
-    ENNOMessagEaseSymbols(96, KB_EN_NO_MESSAGEASE_SYMBOLS),
+    ENMessageEaseLeft(16, KB_EN_MESSAGEEASE_LEFT),
```

Comment:
Revert this change. All new layouts need to be added to the end with a higher index.

Reply:
Ah I see, otherwise updates mess with it. Sure